cmd/collector/app/http_handler.go

```diff
@@ -95,3 +123,45 @@ func (aH *APIHandler) SaveSpan(w http.ResponseWriter, r *http.Request) {
 
 	w.WriteHeader(http.StatusAccepted)
 }
+
+func (aH *APIHandler) ServeSampling(w http.ResponseWriter, r *http.Request) {
```

Comment:
This looks like a copy-paste of the code from the agent. We need to refactor that code into a reusable piece that can be called from both the agent and collector.